packages/ignition/src/app/locales/docs/usage/sizing-elements.pt-br.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Dimensionando Elementos
+title: Dimensionando Ícones
```

Comment:
I prefer call that "Icons", but we must rename the file as well to match the content.